ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-templates.yml

```diff
@@ -32,8 +33,6 @@ default_templates:
     ip_addressing:
       python_module: "ansible_collections.arista.avd.roles.eos_designs.python_modules.ip_addressing"
       python_class_name: "AvdIpAddressing"
-      mlag_ibgp_peering_ip_primary: "ip_addressing/mlag-ibgp-peering-ip-primary.j2"
```

Comment:
These j2 files are no longer needed in the defaults, since it will be covered by the python module referenced above. It is still possible to override with j2 like we have added in the molecule example.